src/main/java/ua/danit/final_project/configuration/StaticCollection.java

```diff
@@ -1,20 +1,23 @@
 package ua.danit.final_project.configuration;
 
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.springframework.beans.factory.annotation.Autowired'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=103&issues=AWW47gZV_eEZPtUmVxCG&open=AWW47gZV_eEZPtUmVxCG)

[ISSUE_KEY]: AWW47gZV_eEZPtUmVxCG